tests/test-term-sync.php

```diff
@@ -386,4 +386,74 @@ public function test_post_trash_to_publish_creates_term_and_restores_relationshi
 
 		$this->assertEquals( array( $term->slug ), $associated_terms, 'Existing shadow term relationships should be restored when its connected post is moved from trash to publish.' );
 	}
+
+	/**
+	 * An existing published post that has its title changed should change the
+	 * title of its shadow term.
+	 */
+	public function test_post_publish_to_publish_modified_title_updates_term() {
+		$post = $this->factory->post->create(
+			array(
+				'post_type'   => 'example',
+				'post_title'  => 'Garbanzo Bean',
+				'post_status' => 'publish',
+			)
+		);
+		$post = get_post( $post );
+
+		$post->post_title = 'Chickpea';
+		wp_update_post( $post );
+
+		$term      = get_term_by( 'slug', 'garbanzo-bean', 'example_connect', OBJECT );
+		$term_name = ! $term ? '' : $term->name;
+
+		$this->assertEquals( 'Chickpea', $term_name, 'A connected term slug should update when a post title is updated.' );
```

Comment:
```suggestion
		$this->assertEquals( 'Chickpea', $term_name, 'A connected term name should update when a post title is updated.' );
```

Reply:
Ugh, I didn't proof these well at all. All of the `assertEquals` are correct—I think. We're always retrieving the term by its slug and then comparing the resulting name. But the error messages are likely wrong! I'll poke through it. Thanks!